CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ## Unreleased
 
 ### Added
+- Vue.js alpha support (#1751)
```

Comment:
Is it experimental support?

Reply:
ah yes. Well it's using the Javascript parser for parsing the patterns, so all the alpha tests should work.